Core/FieldHandler/EzCountry.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Kaliop\eZMigrationBundle\Core\FieldHandler;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\Core\FieldType\Country\Value as CountryValue;
+use eZ\Publish\Core\FieldType\Selection\Value as SelectionValue;
+use Kaliop\eZMigrationBundle\API\FieldValueImporterInterface;
+
+class EzCountry extends AbstractFieldHandler implements FieldValueImporterInterface
```

Comment:
I think we also miss `fieldValueToHash`. Otherwise we will not be able to do a round-trip: export existing content as migration then import it back

Reply:
ok, i'll work on that too

Reply:
as said in our slack conversation, it looks there is no need for fieldValueToHash at first. attribute looks to be exported with an array of the alpha2 codes.